main.tf

```diff
@@ -1,13 +1,192 @@
-resource "random_integer" "example" {
-  count = module.this.enabled ? 1 : 0
+locals {
+  enabled = module.this.enabled
+}
+
+resource "aws_ssm_maintenance_window" "scan_window" {
+  count    = local.enabled ? 1 : 0
+  name     = "module.this.id"
```

Comment:
```suggestion
  name     = module.this.id
```

Reply:
the module creates many `aws_ssm_maintenance_window`.
Use `null-label` to create different names for them

Add `scan_window_label` label module.

Then use

```
name     = module. scan_window_label.id
```